<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mugi-query</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="horizontalSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="verticalSplitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTreeView" name="sessionTree">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
       </widget>
       <widget class="QTreeView" name="schemaTree">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTabWidget" name="sessionTabs">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>&amp;Database</string>
    </property>
    <addaction name="databaseConnect"/>
    <addaction name="databaseReconnect"/>
    <addaction name="databaseDisconnect"/>
    <addaction name="databaseHistory"/>
   </widget>
   <widget class="QMenu" name="menuQuery">
    <property name="title">
     <string>&amp;Query</string>
    </property>
    <addaction name="queryExecute"/>
    <addaction name="queryHistory"/>
    <addaction name="queryHelp"/>
    <addaction name="separator"/>
    <addaction name="queryJoin"/>
    <addaction name="queryQuote"/>
    <addaction name="queryUnquote"/>
   </widget>
   <widget class="QMenu" name="menuSession">
    <property name="title">
     <string>&amp;Session</string>
    </property>
    <addaction name="sessionAdd"/>
    <addaction name="sessionRemove"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>&amp;Data</string>
    </property>
    <addaction name="dataSave"/>
    <addaction name="dataImport"/>
   </widget>
   <widget class="QMenu" name="menuSelection">
    <property name="title">
     <string>S&amp;election</string>
    </property>
    <addaction name="selectionCopy"/>
    <addaction name="selectionCopyAsList"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="settingsFormat"/>
   </widget>
   <addaction name="menuDatabase"/>
   <addaction name="menuSession"/>
   <addaction name="menuQuery"/>
   <addaction name="menuData"/>
   <addaction name="menuSelection"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="databaseConnect">
   <property name="text">
    <string>&amp;Connect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveData">
   <property name="text">
    <string>&amp;Save data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="databaseHistory">
   <property name="text">
    <string>&amp;History</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="queryHistory">
   <property name="text">
    <string>&amp;History</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="sessionAdd">
   <property name="text">
    <string>&amp;Add</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="sessionRemove">
   <property name="text">
    <string>&amp;Remove</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="databaseDisconnect">
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="databaseReconnect">
   <property name="text">
    <string>Re&amp;connect</string>
   </property>
  </action>
  <action name="dataSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="dataCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="queryHelp">
   <property name="text">
    <string>He&amp;lp</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="queryCreateUser">
   <property name="text">
    <string>Create user</string>
   </property>
  </action>
  <action name="queryJoin">
   <property name="text">
    <string>&amp;Join</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="dataCopyAsList">
   <property name="text">
    <string>Copy as list</string>
   </property>
  </action>
  <action name="selectionCopy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="selectionCopyAsList">
   <property name="text">
    <string>Copy as &amp;list</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="queryQuote">
   <property name="text">
    <string>&amp;Quote</string>
   </property>
  </action>
  <action name="queryUnquote">
   <property name="text">
    <string>&amp;Unquote</string>
   </property>
  </action>
  <action name="settingsFormat">
   <property name="text">
    <string>Format</string>
   </property>
  </action>
  <action name="dataPlot">
   <property name="text">
    <string>&amp;Plot</string>
   </property>
  </action>
  <action name="queryExecute">
   <property name="text">
    <string>&amp;Execute</string>
   </property>
  </action>
  <action name="queryTables">
   <property name="text">
    <string>&amp;Tables</string>
   </property>
  </action>
  <action name="dataImport">
   <property name="text">
    <string>&amp;Import</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
